fetch-in-react/my-app/src/User.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import UserCard from './UserCard';
+
+export default function User({ userId, onCancel }) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+  const [user, setUser] = useState();
+
+  useEffect(() => {
+    async function listUserId() {
+      try {
+        const res = await fetch(`https://jsonplaceholder.typicode.com/users/${userId}`);
+        const jsonData = await res.json();
+        if (!res.ok) {
+          throw new Error(`error ${res.status}`);
+        } else {
+          setUser(jsonData);
+        }
+      } catch (err) {
+        console.error(err);
```

Comment:
Set the error state so it will display on the page.

Reply:
it seems my save didn't push resent just now.